Gopkg.toml

```diff
@@ -6,7 +6,7 @@
 
 [[constraint]]
   name = "github.com/gorilla/mux"
-  version = "1.6.1"
+  version = "=1.6.1"
```

Comment:
Backwards-incompatible change is introduced in mux 1.6.2 - https://github.com/gorilla/mux/pull/338 
which prevents the router from starting due to https://github.com/pmorie/osb-broker-lib/blob/master/pkg/server/server.go#L63

We should either stick to 1.6.1 or ignore the error for the `healthz` endpoint

Reply:
can you add this as a comment above the constraint in the toml file?